etl/steps/data/garden/who/2023-04-03/flu_vaccine_policy.py

```diff
@@ -0,0 +1,92 @@
+"""Load a meadow dataset and create a garden dataset."""
+import numpy as np
+import pandas as pd
+from owid.catalog import Dataset, Table
+from structlog import get_logger
+
+from etl.data_helpers import geo
+from etl.helpers import PathFinder, create_dataset
+
+log = get_logger()
+
+# Get paths and naming conventions for current step.
+paths = PathFinder(__file__)
+
+
+def run(dest_dir: str) -> None:
+    log.info("flu_vaccine_policy.start")
+
+    #
+    # Load inputs.
+    #
+    # Load meadow dataset.
+    ds_meadow: Dataset = paths.load_dependency("flu_vaccine_policy")
+
+    # Read table from meadow dataset.
+    tb_meadow = ds_meadow["flu_vaccine_policy"]
+
+    # Create a dataframe with data from the table.
+    df = pd.DataFrame(tb_meadow).reset_index()
+
+    #
+    # Process data.
+    #
+    log.info("flu_vaccine_policy.harmonize_countries")
+    df = geo.harmonize_countries(
+        df=df, countries_file=paths.country_mapping_path, excluded_countries_file=paths.excluded_countries_path
+    )
+    # Replacing value codes with either missing data or a more descriptive value
+    df = df.replace({"ND": np.NaN, "NR": "Not relevant", "Unknown": np.NaN})
+    # Removing strings from some values e.g. commas in numbers
+    df["how_many_doses_of_influenza_vaccine_were_distributed"] = df[
+        "how_many_doses_of_influenza_vaccine_were_distributed"
+    ].str.replace(r"[^0-9]", "", regex=True)
+
+    df = clean_binary_colums(df)
+    df = clean_hemisphere_formulation(df)
+    # Removing erroneous value from is_influenza_vaccination_recommended_for_children
+    df["is_influenza_vaccination_recommended_for_children"].replace(0, np.NAN)
```

Comment:
Just guessing: Couldn't "0" rather mean "No" instead of missing?

Reply:
I think as it only occurs a couple of times I wouldn't feel comfortable assigning it a value.